extensions/.eslintrc.js

```diff
@@ -0,0 +1,38 @@
+module.exports = {
```

Comment:
Won't limiting Jest config to this folder make it harder to migrate from Mocha to Jest outside `/extensions`?

Reply:
I don't think it will make it harder… the hard part will be migrating the tests. Merging/cleaning the eslint will hopefully be the easy part.

The eslint setup is already problematic… too many globals are enabled on too many files. I really think we need 2 eslint configs right now:
- `_inc/client` and `extensions` contain sources which will be compiled. Those need specific rules.
- Everything else is just JS that will be served as-is. They require a completely different set of rules, browser-only and es5.

I spent a lot of time trying to detangle all of this a few days ago and it's very confusing. It's also very difficult to get the overrides right because they either aren't all applied or aren't merged in the way you'd expect.

This needs to be revisited from the ground up.

Reply:
>I really think we need 2 eslint configs right now:

Yup, makes sense. I had a look around as well and agree it gets pretty confusing really fast. Without any serious refactoring, Jetpack might need 4 eslint configs to get rid of jshint. 😱 

>It's also very difficult to get the overrides right because they either aren't all applied or aren't merged in the way you'd expect.

Ditto.

Thanks for elaborating!